AUTHORS

```diff
@@ -1314,3 +1314,4 @@ Alberto Jiménez Ruiz <Alberto.Jimenez@uclm.es>
 João Bravo <joaocgbravo@tecnico.ulisboa.pt>
 Dean Price <dean1357price1357@gmail.com>
 Hugo Kerstens <hugo@kerstens.me>
+Jatin Bhardwaj <bhardwajjatin@gmail.com>
```

Comment:
You need to rerun the `bin/mailmap_check.py` script to sort the .mailmap file.

Reply:
can you please guide now how i approach the isssue

Reply:
As I said you need to run the script.